news/6413.bugfix

```diff
@@ -0,0 +1,3 @@
+Fix pip install <url> allow directory traversal when a malicious server
+(or a network MitM if downloading over HTTP) send a Content-Disposition header
+with filename which contains "../ or ..\\".
```

Comment:
Here is suggested rewording of what you wrote to fix some wording issues:

Prevent `pip install <url>` from permitting directory traversal if e.g. a malicious server sends a Content-Disposition header containing a filename with `"../"` or `"..\\"`.

(Also, be sure to enclose the "code" strings with _double_ backticks, both before and after.)